### ISSUE TITLE
Time series example

### ISSUE THREAD

**Issue by jebyrnes** [JenniNiku/gllvm#42]
Hello! Still loving this package. I was wondering if you ever put together/had a good example of a multivariate time series? e.g., to see if some external driver affected community structure over time.

(and if you are looking for one, I've got a great example!)

**Comment by JenniNiku** [comment]
Hi! Nice to hear that. I have started working on such model that could take into account the time dependency in the data, so at least something related to that might come sometime in a few months.

**Comment by jebyrnes** [comment]
That's awesome! If you are interested, I have access to a 30 year quarterly sampled fixed quadrat (n=45) study with ~150 species, if you are interested in looking that as a demo dataset. I've also got access to a shorter-term LTER dataset of ~100 species at fixed transects from 2000-2020 in kelp forests sampled annually. If that's of interest, feel free to email me or let me know here!

**Comment by JenniNiku** [comment]
Yeah. I think we have rather short time series data at the moment, so those datasets 
 you mentioned sounds very interesting.

**Comment by JonJup** [comment]
Hi, I was wondering whether you made progress on the time series aspect. 
Thanks !

**Comment by BertvanderVeen** [comment]
Hello Jonathan. Structured row effects are now implement in the package, see #1, which allow for an AR1 random row effect (see `row.eff` in `gllvm(.)`). @JenniNiku is working on functionality for structured latent variables, but (I think) that is not ready yet.

**Comment by JonJup** [comment]
Thanks Bert for the quick answer! Is it possible to have a combined structured/ unstructured row effect? We have time series data from different mesocosms. I have gotten the model working with a corAR1 row effect for time, but dont know if I can still add a random effect for the mesocosm now.

**Comment by BertvanderVeen** [comment]
I'm afraid not. At present gllvm is limited to a single random row effect. Could you consider having an additional fixed row effect instead?

**Comment by JonJup** [comment]
Thanks again! Yes thats what we will porbably do in that case.

**Comment by rialsc** [comment]
> Hello Jonathan. Structured row effects are now implement in the package, see #1, which allow for an AR1 random row effect (see `row.eff` in `gllvm(.)`). @JenniNiku is working on functionality for structured latent variables, but (I think) that is not ready yet.

Hello! Ive looked through the documentation and many examples, Do you have an example of how to write this out? Ive tried different variations of this but it hasnt worked for me:
 row.eff = corAR1(~1| Age_myr)

Thank you!

**Comment by BertvanderVeen** [comment]
> > Hello Jonathan. Structured row effects are now implement in the package, see #1, which allow for an AR1 random row effect (see `row.eff` in `gllvm(.)`). @JenniNiku is working on functionality for structured latent variables, but (I think) that is not ready yet.
> 
> Hello! Ive looked through the documentation and many examples, Do you have an example of how to write this out? Ive tried different variations of this but it hasnt worked for me:
>  row.eff = corAR1(~1| Age_myr)
> 
> Thank you!

Sorry about that, the vignette is still under construction. `row.eff=~corAR1(1|time)` should do it I'd think!

**Comment by jebyrnes** [comment]
Aaaaamazing! Trying it right now with another data set!

I'm still wrapping my head around ordination - is it possible to extract axes and determine what they indicate, then plot change in ordination space through time? I'm guessing yes?

I have a student who is trying to look at changes in dominant species/composition over time and this seems perfect. But trying to figure out how to get such information from a timeseries model here with time as a predictor...
